Movie_App.js

```diff
@@ -0,0 +1,128 @@
+document.addEventListener("DOMContentLoaded", () => {
+    fillSelectTag();
+    getIds();
+
+    window.addEventListener("load", start, false);
+
+    let form1 = document.querySelector('#form')
+    form1.addEventListener('submit', (event) => {
+        event.preventDefault();
+        let review = document.querySelector('#review');
+        // let newItem = document.querySelector('#new_item');
+
+        let reviewLocation = document.querySelector('#submited-reviews');
+        let newReview = document.createElement("li");
+
+        let bold = document.createElement("B");
+        let title = document.createTextNode(currentMovie());
+
+        let userReview = document.createTextNode(": " + review.value);
```

Comment:
Just for reference, another way of getting review.value would be this "event.target.childNodes[1].value". No major different between them but event.target can get handy in some occasions.